package.json

```diff
@@ -138,13 +147,13 @@
     "terra-search-field": "^3.13.0",
     "terra-spacer": "^3.3.0",
     "terra-status-view": "^4.1.0",
+    "terser-webpack-plugin": "^1.1.0",
     "to-ast": "^1.0.0",
     "webpack-merge": "^4.1.2"
   },
   "peerDependencies": {
     "react-dom": "^16.8.5",
     "react": "^16.8.5",
-    "terra-toolkit": "^5.21.0 || ^6.0.0",
     "webpack": "^4.28.1"
```

Comment:
you will need to add some more peerDependencies... While it is normally not passive to add peer dependencies, In this case it is fine since we're adding back the peer dependencies a previous peer dependency would have added.

```json
    "@babel/cli": "^7.4.4",
    "@babel/core": "^7.4.4",
    "core-js": "^3.1.3",
    "raf": "^3.4.1",
    "regenerator-runtime": "^0.13.2",
    "webpack-cli": "^3.3.2",
```

Reply:
Updated here https://github.com/cerner/terra-dev-site/pull/337/commits/d7028ef5ae602185c64c57811c6f1d7c47ef126c